### DISCUSSION TITLE
How do I use the combos from the full list combo? (I'm new here)

### DISCUSSION THREAD

**Discussion by nogz01** [PrincessRTFM/XIVComboPlugin#39] [Q&A]
Hi, I'm new to this ffxiv plugin world, I was able to install the ffxiv launcher and the ffxivComboPlugin succesfully.
I saw this page: https://github.com/PrincessRTFM/XIVComboPlugin#readme
This list shows way more combo options than what XIVComboPlugins have, how do I use those?
I already put the "Vixen's Dalamud Plugins" code in the Experimental tab, what am I doing wrong?

![122](https://user-images.githubusercontent.com/103921700/163768338-6b5b9f5e-479b-480e-bc38-45ef275436d4.png)

**Comment by VariableVixen** [comment]
Only use one combo plugin at a time, first of all. Otherwise, they can conflict and weird things might happen, plus you won't get support from _any_ of their devs.

After putting the URL into your list, hit save and close, then open your plugin installer window (`/xlplugins`) to remove all versions except the **one** (1) you will use. Install that one, and _only_ that one, then use `/pcombo` to open the settings for it. Configure it as you like.

**Reply by nogz01** [reply]
Hello Princess, many thanks for the reply.
I realised that I wasn't going back to the plugin installer window, I thought that putting the URL would give me the extended combos.
Now it's working perfectly, it's a game changer, my Dragoon feels amazing!

I have a suggestion, something like "Use Full Thrust combo, if Disembowel buff is not present then use Disembowel combo".
Do you think something like that would work? 

Thanks for your help.

**Reply by VariableVixen** [reply]
Please put combo requests into a [new issue on the tracker](https://github.com/PrincessRTFM/XIVComboPlugin/issues/new) for easier tracking. However, at a rough glance, that would probably be doable.
